packages/astro-ghostcms/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import astroghostcms from "./astro-ghostcms";
+
+export default astroghostcms;
```

Comment:
Not a huge fan of this not being camel cased

Reply:
fixed... idk how that even happened... lol that must have been done by VSCode when i changed the name of the file 🤣